e2e/session.test.ts

```diff
@@ -0,0 +1,311 @@
+    // move to start page
+    await getMenuItem(page, 'start').click();
+    await expect(page).toHaveURL(/\/start/);
+    // click Start Session button and move to session start page
+    const card = getCardItemByCardTitle(page, 'Start Batch Session');
+    expect(card).toBeVisible();
+    const creationButton = card.getByRole('button', {
+      name: 'Start Session',
+    });
+    await creationButton.click();
+    await expect(page).toHaveURL(/\/session\/start/);
+    await page.waitForLoadState('networkidle');
+    // batch radio button is selected by default
+    const batchRadioButton = page
+      .locator('label')
+      .filter({ hasText: 'Batch' })
+      .locator('input[type="radio"]');
+    await expect(batchRadioButton).toBeChecked();
+    // create session
+    await createBatchSessionOnSessionStartPage(page, sessionName);
+    // Verify that a cell exists to display the session name
+    const session = page
+      .locator('vaadin-grid-cell-content')
+      .filter({ hasText: `${sessionName} edit done` });
+    // it takes time to show the created session
+    await expect(session).toBeVisible({ timeout: 10000 });
+  });
+
+  test('User can create interactive session on the Sessions page', async ({
+    page,
+  }) => {
+    // move to sessions page
+    await getMenuItem(page, 'sessions').click();
+    await expect(page).toHaveURL(/\/job/);
+    // click start button to create session
+    const startButton = getStartSessionButton(page);
+    expect(startButton).toBeVisible();
+    await startButton.click();
+    await expect(page).toHaveURL(/\/session\/start/);
+    await page.waitForLoadState('networkidle');
+    await createInteractiveSessionOnSessionStartPage(page, sessionName);
+    // close app dialog
+    await page.getByRole('button', { name: 'close' }).click();
+    // Verify that a cell exists to display the session name
+    const session = page
+      .locator('vaadin-grid-cell-content')
+      .filter({ hasText: `${sessionName} edit done` });
+    // it takes time to show the created session
+    await expect(session).toBeVisible({ timeout: 10000 });
+  });
+
+  test('User can create batch session on the Sessions page', async ({
+    page,
+  }) => {
+    // move to sessions page
+    await getMenuItem(page, 'sessions').click();
+    await expect(page).toHaveURL(/\/job/);
+    // click start button to create session
+    const startButton = getStartSessionButton(page);
+    expect(startButton).toBeVisible();
+    await startButton.click();
+    await expect(page).toHaveURL(/\/session\/start/);
+    await page.waitForLoadState('networkidle');
+
+    await createBatchSessionOnSessionStartPage(page, sessionName);
+    // Verify that a cell exists to display the session name
+    const session = page
+      .locator('vaadin-grid-cell-content')
+      .filter({ hasText: `${sessionName} edit done` });
+    // it takes time to show the created session
+    await expect(session).toBeVisible({ timeout: 10000 });
```

Comment:
Batch session is automatically terminated after completing the startup command. So, I got `Timed out 10000ms waiting for expect(locator).toBeVisible()` error message because it is already terminated. Let's find the session in other ways.